src/controllers/dynakube/activegate/reconciler.go

```diff
@@ -0,0 +1,139 @@
+package activegate
+
+import (
+	"context"
+	"fmt"
+
+	dynatracev1beta1 "github.com/Dynatrace/dynatrace-operator/src/api/v1beta1"
+	"github.com/Dynatrace/dynatrace-operator/src/dtclient"
+	"github.com/Dynatrace/dynatrace-operator/src/kubeobjects"
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+const (
+	AGSecretSuffix             = "-activegate-tenant-secret"
+	CommunicationEndpointsName = "communication-endpoints"
+	TenantTokenName            = "tenant-token"
+	TenantUuidName             = "tenant-uuid"
```

Comment:
```suggestion
	TenantUUIDName             = "tenant-uuid"
```

Reply:
It's less readable imho. 
No two capitalised letters shall follow directly each other in cammelCase.

Reply:
uh, spicy 🌶️  :D 
I would disagree 😏

Reply:
I think we need a guideline note about preferred coding style with regard to camel casing. I remember refactoring a lot of code and APIs to follow `StatsdSomething` instead of `StatsDSomething`. 

You're now suggesting the opposite 😛

Reply:
Very true 😅 

I just saw a lot of `SomethingID` and `SomethingUUID` in our code so I just wanted to be consistent (that said there is also `SomethingId` and `SomethingUiid` in our code 😛 )

Reply:
@toszr why the opposite? I stick exactly to it by doing `UuidName` instead of `UUIDName` :open_mouth:.
Naming fight, start! :smile:

Reply:
Hmm, my comment was to the Marcell's suggestion but apparently I was too slow to post it. So, to clarify:
- I was advised a while ago by @mjgrzybek to switch convention from `StatsDSomething` to `StatsdSomething`
- @0sewa0 is now suggesting to switch from `TenantUuidName` to `TenantUUIDName`

That is why I suggested we have a guideline to simplify reviews.